internal/tiltfile/tiltfile_state.go

```diff
@@ -815,6 +819,15 @@ func (s *tiltfileState) translateK8s(resources []*k8sResource) ([]model.Manifest
 		}
 
 		result = append(result, m)
+		for _, iTarg := range iTargets {
+			ref := iTarg.ConfigurationRef.String()
+			hasLiveUpd := fmt.Sprintf("%t", !iTarg.AnyLiveUpdateInfo().Empty())
+			count := r.imageRefMap[ref]
+
+			a.Count("containersForRef", map[string]string{
```

Comment:
re: `containersForRef` -- do we have any casing guidelines for metric names? 🤷‍♀

Reply:
this is a bad place to report this data, because it will get executed every time the tiltfile executes. please move this kind of reporting to the engine/analytics_reporter. That is what it exists for.

Reply:
I did this to avoid storing extra data on the manifest b/c we currently don't know how many times a given image target is referenced in the yaml (unless we re-parse the yaml) -- but sure, can do.